.github/actions/verify-token/dist/index.js

```diff
@@ -88,8 +88,15 @@ function run() {
             const workflowRecipient = core.getInput("slsa-workflow-recipient");
             const unverifiedToken = core.getInput("slsa-unverified-token");
             const outputPredicate = core.getInput("output-predicate");
+            if (!outputPredicate) {
+                // detect if output predicate is null or empty string.
+                throw new Error("output-predicate must be supplied");
+            }
             const wd = (0, utils_1.getEnv)("GITHUB_WORKSPACE");
             const safeOutput = (0, utils_1.resolvePathInput)(outputPredicate, wd);
+            if (fs.existsSync(safeOutput)) {
```

Comment:
maybe you can create the function to address https://github.com/slsa-framework/slsa-github-generator/issues/1513 as part of the PR, like `safeWrite()`. Just a thought, not really necessary

Reply:
Yes, I'm working on creating common utils now -- including a common file utility (resolving paths, safeWrite). I'll update that todo